src/notificationcenter.cs

```diff
@@ -47,6 +64,107 @@ interface UIVibrancyEffect_NotificationCenter {
 		[Static, Export ("notificationCenterVibrancyEffect")]
 		UIVibrancyEffect NotificationCenterVibrancyEffect ();
 	}
+#endif
+
+#if MONOMAC
+	[Mac (10,10)]
+	[BaseType (typeof(NSViewController), Delegates=new string [] { "Delegate" }, Events=new Type [] { typeof (NCWidgetListViewDelegate)})]
+	interface NCWidgetListViewController
+	{
+		[Wrap ("WeakDelegate")]
+		[Protocolize]
```

Comment:
Don't use [Protocolize] in new code, just bind using the protocol (`INCWidgetListViewDelegate`) instead.